test/unit/email_rate_limit_test.rb

```diff
@@ -13,9 +9,9 @@ def seed_array
     invalid_values        = last_clicked_days_ago.to_a - valid_values
     day_ago               = rand(last_clicked_days_ago)
     clicked_ago           = valid_values.sample
-    assert EmailRateLimit.new(day_ago).now?(clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).now?(#{clicked_ago}) to be true, was not"
+    assert_true { EmailRateLimit.new(day_ago).now?(clicked_ago) }
```

Comment:
> assert_equal true, foo?, the error messages appear to be pretty useless.

I think if that's what you want, you should just print the whole line, like RSpec does.

> Failure/Error: expect(@breadcrumbs.empty?).to be false
> expected false
> got true

There's probably already a minitest formatter that does this.